Match screen

FrederikSchlemmer opened the pull request:
This branch is based on the starting-screen-menu branch.

Fixes #31

FrederikSchlemmer committed:
Feature: Match Screen

FrederikSchlemmer committed:
Added MatchScreen

Branch is based on the starting-screen-menu Branch.

FrederikSchlemmer committed:
Added logo to start menu

FlorianTim commented on core/src/de/adesso/brainysnake/BrainySnake.java:
```diff
@@ -6,44 +6,83 @@
+        createBasicSkin();
+
+        try {
+            logoTexture = new Texture(Gdx.files.internal("./core/src/de/adesso/brainysnake/img/BrainySnake_Headline.png"));
```
Ich glaube das hatten wir schon?
Bitte in den assets ordner

FrederikSchlemmer replied:
Im aktuellen Branch ist das normalerweise schon bereits behoben. Basiert der Kommentar vielleicht auf einem älteren Commit?

RudiKlassen-zz replied:
@FrederikSchlemmer Die Requests beziehen sich immer auf den Stand, der zum Zeitpunkt des Reviews auf dem Branch herrscht. Wenn auf dem PR noch weitere Pushes folgen, kommt es zu obigen Verwirrungen, was ich aber nicht weiter wild finde. Evtl. müssen wir da noch etwas justieren.

FlorianTim commented on core/src/de/adesso/brainysnake/BrainySnake.java:
```diff
@@ -6,44 +6,83 @@
+            logoBrainySnake = new Image(region);
+            logoBrainySnake.setPosition(Config.APPLICATION_WIDTH / 4, Config.APPLICATION_HEIGHT - 250);
+        } catch (Exception e){
+            LOGGER.error(e.toString());
```
Die genaue Exception fangen
Dem Logger nur e übergeben

FrederikSchlemmer replied:
Dieser Try/Catch ist ebenfalls im aktuellen Branch nicht mehr vorhanden.

FlorianTim commented on core/src/de/adesso/brainysnake/BrainySnake.java:
```diff
@@ -101,23 +141,124 @@ private void initializeCamera() {
+        stage.draw();
     }
 
+    private void createBasicSkin() {
```
JavaDoc

FrederikSchlemmer replied:
Erledigt.

FlorianTim commented on core/src/de/adesso/brainysnake/BrainySnake.java:
```diff
@@ -101,23 +141,124 @@ private void initializeCamera() {
         fontCamera.setToOrtho(false, APPLICATION_WIDTH, APPLICATION_HEIGHT);
     }
 
+    public void drawMatchScreen(){
```
Das können wir vielleicht in mehrere Methoden aufteilen
+JavaDoc

FrederikSchlemmer replied:
Aufgeteilt in mehrere Methoden und JavaDoc hinzugefügt.

FrederikSchlemmer committed:
DrawMatchScreen divided in multiple methods

Divided functionality of DrawMatchScreen in multiple methods. Added JavaDoc to methods.

RudiKl committed:
Merge branch 'master' into match-screen

RudiKl committed:
Fix merge - error
